Story/0AllStory.cs

```diff
@@ -56,6 +50,12 @@
 
 //cs_include Scripts/Story/ThroneofDarkness/CoreToD.cs
 
+//cs_include Scripts/Story/ShadowsOfChaos/CoreSoC.cs
+
+//cs_include Scripts/Story/FireIsland/CoreFireIsland.cs
+
+//cs_include Scripts/Story/ShadowsOfChaos/CoreSoC.cs
```

Comment:
duplicate include: //cs_include Scripts/Story/ShadowsOfChaos/CoreSoC.cs

Reply:
yes mb this was meant for //cs_include Scripts/Story/DageTheEvilIsland/CoreDageTheEvilIsland.cs